.github/ISSUE_TEMPLATE/02_feature_request.yml

```
@@ -0,0 +1,94 @@
+      label: Contribution
+      description: Would you be willing to contribute to this feature?
+      options:
+        - label: I'm willing to submit a PR for this feature
+
+  - type: textarea
+    id: additional-context
+    attributes:
+      label: Additional context
+      description: Add any other context, mockups, or screenshots about the feature request here.
+    validations:
+      required: false
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)